Templates/AzureDevOpsPipeline/templates/tagging/createReleaseCandidate.yml

```
@@ -1,138 +1,106 @@
-            echo $ERRMSG
+            ERRMSG="[ERROR] The log file '${localComputeReleaseVersionLog}' was not found. Please inspect azure runner environment and log output. rc="$rc
+            echo "${PGM}: $ERRMSG"
           fi
-          ##DEBUG ## echo -e "baselineRef \t: ${baselineRef}"    ## DEBUG
-        fi
-
-        # echo baseline Tag = ${baselineRef} # Debug
 
-        #-2- computes the name of the next release based on the pipeline parameter `releaseType` (major, minor, patch)
-        if [ $rc -eq 0 ]; then
-          # echo "create new release candidate" # Debug
-          if [ "$(releaseType)" == "patch" ]; then
-            export newVersionTag=$(echo ${baselineRef} | sed 's/^["refs\/tags\/rel-]*//g' | sed 's/-[a-zA-Z0-9]*//g' | awk -F. -v OFS=. '{$3 += 1 ; print}')
+          # Check nextReleaseVersion variable retrieved by sed
+          if [ $rc -eq 0 ]; then
+            if [ -z "${nextReleaseVersion}" ]; then
+              rc=8
+              ERRMSG="[ERROR] The name of the next release could not be parsed. Check stage and $(Build.ArtifactStagingDirectory)/computeReleaseVersion on the ADO runner environment. rc="$rc
+              echo "${PGM}: $ERRMSG"
+            else
+              echo "${PGM}: [INFO] Retrieved next release version: $nextReleaseVersion"
+              echo "##vso[task.setvariable variable=nextReleaseVersion;isOutput=true]$nextReleaseVersion"
+            fi
           fi
-          if [ "$(releaseType)" == "minor" ]; then
-            export newVersionTag=$(echo ${baselineRef} | sed 's/^["refs\/tags\/rel-]*//g' | sed 's/-[a-zA-Z0-9]*//g' | awk -F. -v OFS=. '{$2 += 1 ; $3 = 0; print}')
-          fi
-          if [ "$(releaseType)" == "major" ]; then
-            export newVersionTag=$(echo ${baselineRef} | sed 's/^["refs\/tags\/rel-]*//g' | sed 's/-[a-zA-Z0-9]*//g' | awk -F. -v OFS=. '{$1 += 1 ; $2 = 0; $3 = 0; print}')
-          fi
-          export newVersionTag=$(echo "rel-${newVersionTag}")
-          echo "${PGM}: [INFO] Name of next planned release is ${newVersionTag}"
-          # Store a the newVersionTag name as an ADO variable 
-          # https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-a-multi-job-output-variable
-          echo "##vso[task.setvariable variable=nextReleaseName;isOutput=true]${newVersionTag}"
-        fi
 
-        #-3- Compute the name of the next release candidate
-        if [ $rc -eq 0 ]; then
-          # Fetch exiting release candidate via AZ cli and some grep operations
-          # validates if a release candidate exits (using AZ CLI)
-          releaseCandidate=$(az repos ref list --org $(System.CollectionUri) --project $(System.TeamProjectId) -r $(application) --filter tags --query '[?name].name | sort(@)' | grep ${newVersionTag}_rc | tail -n 1)
+          # Set new Git tag
+          if [ $rc -eq 0 ]; then
+
+            newVersionTag="${nextReleaseVersion}.$(buildNumber)"
```

```suggestion
            newVersionTag="${nextReleaseVersion}-$(buildNumber)"
```
Using a dash instead of a dot?